pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1356,6 +1356,28 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
     $(document).ready(function() {
        $('.modal').on('shown.bs.modal', function () {
             $('.btn-default').focus();
+            $(this).find(".modal-body").attr("tabindex", 0);
        });
-    })
+
+        // make it possible to tab to response content inside popups for keyboard users
+        $(".modal-body pre").attr("tabindex", 0);
+        $(".modal-body").attr("aria-label", "Response body");
+        
+        // hack to get the screen reader to read the syntax-highlighted
+        // code section on Microsoft Edge
+        document.querySelectorAll(".modal-body").forEach((modalBody) => {
```

Comment:
document.querySelectorAll has some inconsitencies in older browsers and across platforms. 
Since we already have jQuery do use $('.modal-body') to select the elements so we support a huge number of devices. Do note that Jquery will use queryselectorall when its available. so the perfomance hit should be minimal